File: MT.Model/Dictionary/Term.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region Term:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a Term:EntityBase.
    /// </summary>
    public class Term : EntityBase
    {
        [Key]
        public int TermID { get; set; }
        public string TermCode { get; set; }
        public string TermName { get; set; }
        public string Description { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }
    }
    #endregion
}

File: BECore/app/model/Term.js
/// <summary>
/// Model EXTJS Degree.
/// </summary>
Ext.define('QLDT.model.Term', {
    extend: 'Ext.data.Model',

    idProperty: 'TermID',
    idType: 'int',

    code: 'TermCode',
    name: 'TermName',

    fields: [
        { name:"TermCode", type:"string" },
        { name:"TermName", type:"string" },
        { name:"Description", type:"string" },
        { name:"Inactive", type:"boolean" },
        { name:"SortOrder", type:"int" }
     ]
});

